src/_help/continuous-integration/cli.md

```diff
@@ -226,6 +226,27 @@ _Note: the additional `--open` flag helps to automatically open the preview URL
 
 Please check `bump preview --help` for more usage details
 
+### `bump overlay [FILE1] [FILE2] > [FILE3]`
+
+> This feature is only compatible with OpenAPI definition files, and support the [Overlay specification](https://github.com/OAI/Overlay-Specification).
```

Comment:
This is not true :) you can use `bump overlay` to apply OAI Overlays to an AsyncAPI definition file :)